README.md

```diff
@@ -77,143 +72,160 @@ Among other things, you will find in this repository instructions on how to simu
-
-###### ROS 2.0
-
-###### Terminal 1:
-
-You can run `gazebo`, spawn the model, publish the robot state and run `hros_cognition_mara_components` using the ROS 2.0 launch file. You can choose one of the following `ros2 launch` depends on the gripper that you want to use:
+**Optionally**, you can launch one of these other launch files, which correspond to other grippers.
 
 ```
-source ~/ros2_mara_ws/install/setup.bash
-ros2 launch mara_gazebo mara.launch.py
 ros2 launch mara_gazebo mara_gripper_140.launch.py
 ros2 launch mara_gazebo mara_gripper_85.launch.py
 ros2 launch mara_gazebo mara_gripper_hande.launch.py
 ```
 
-###### Terminal 2:
+#### Terminal 2 (ROS):
+```
+source ~/catkin_mara_ws/devel/setup.bash
+
+python3 ~/catkin_mara_ws/src/mara_camera/mara_bringup/scripts/follow_joints_trajectory_actions.py ~/catkin_mara_ws/src/mara_camera/mara_bringup/config/motors.yaml &
+# change the prefix to match with the gripper used in the Terminal 1
+roslaunch mara_bringup mara_bringup_moveit_actions.launch prefix:=140 &
 
-We need to run this node to create a bridge bewteen ROS and ROS 2.0. The topics that will be available are `/mara_controller/state`, `/joints_state` and `hros_actuation_servomotor_*********/trajectory`. Type the following command to run the bridge:
+sleep 4
+roslaunch mara_moveit_config mara_moveit_planning_execution.launch
 
 ```
-source /opt/ros/melodic/setup.bash
+
+#### Terminal 3 (bridge):
+```sh
+source ~/catkin_mara_ws/devel/setup.bash
 source ~/ros2_mara_ws/install/setup.bash
-ros2 run individual_trajectories_bridge individual_trajectories_bridge -motors `ros2 pkg prefix individual_trajectories_bridge`/share/individual_trajectories_bridge/motors.yaml
+
+ros2 run individual_trajectories_bridge individual_trajectories_bridge_actions -motors ~/ros2_mara_ws/src/mara/individual_trajectories_bridge/config/motors_actions.yaml
 ```
 
-###### ROS
+### Usage on the real robot
 
-You can run the four needed nodes using this launch file. You should use the argument `prefix` to indicate which gripper you are using `85`, `140` or `hande`:
+#### Terminal 1 (ROS 2.0)
+```sh
+source ~/ros2_mara_ws/install/setup.bash
+# you will need to change the export values according to the SOMs configuration
+export RMW_IMPLEMENTATION=rmw_opensplice_cpp
+export ROS_DOMAIN_ID=22
 
+ros2 launch mara_bringup mara_bringup_real.launch.py
 ```
-source ~/ros_mara_ws/install_isolated/setup.bash
-roslaunch mara_bringup mara_bringup_moveit.launch prefix:=85
+
+#### Terminal 2 (ROS)
+```sh
+source ~/catkin_mara_ws/devel/setup.bash
+# you will need to change the yaml files to match the topics names on your SOMs
+python3 ~/catkin_mara_ws/src/mara_camera/mara_bringup/scripts/follow_joints_trajectory_actions.py ~/catkin_mara_ws/src/mara_camera/mara_bringup/config/motors.yaml &
+roslaunch mara_bringup mara_bringup_camera_moveit_actions.launch prefix:=140 yaml:=motors_camera.yaml &
+
+sleep 3
+roslaunch mara_moveit_config mara_moveit_planning_execution.launch
```

Comment:
this is not need it. mara_bringup_camera_moveit_actions.launch` should launches this

Reply:
I will test if it works without that.

Reply:
I can not test it because i do not have the robot available but looking at the code you are right. I will also replace the launch file which includes removing yaml arg